nix/pwndbg.nix

```diff
@@ -71,13 +51,18 @@ let
       mkdir -p $out/share/pwndbg
 
       cp -r gdbinit.py pwndbg $out/share/pwndbg
+      # Build self-contained init script for lazy loading from vanilla gdb
+      # I purposely use insert() so I can re-import during development without having to restart gdb
+      sed "2 i import sys, os\n\
+      sys.path.insert(0, '${pyEnv}/${pyEnv.sitePackages}')\n\
+      sys.path.insert(0, '$out/share/pwndbg/')\n\
```

Comment:
`sys.path.insert(0, '$out/share/pwndbg/')`
🤔  why is needed?

Reply:
`pyEnv` only provides the dependencies of pwndbg, whereas for `gdbinit.py` to `import pwndbg`, it needs to import from `share/pwndbg/` as that is where the `pwndbg` python module is installed to.

This is without that line:

```
(gdb) source result/pwndbg/share/gdbinit.py
result/pwndbg/share/gdbinit.py: No such file or directory.
(gdb) source result/share/pwndbg/gdbinit.py
Traceback (most recent call last):
  File "result/share/pwndbg/gdbinit.py", line 65, in <module>
    import pwndbg  # noqa: F401
    ^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'pwndbg'
```

If there's some better way, happy to change it.

Reminds me even though `pwndbg.nix` links the `.venv/` folder, there is no activation script created by the poetry derivation in `.venv/bin/`. Is anyone actually using the venv on nix? Since it won't include pwndbg (as it only links the poetry sitePackages) I think it might not work? Maybe I'm missing something.